app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,56 @@
+
+<%= render 'partials/navigation' %>
+
+<h1><%= @recipe.name%></>
+
+<section id="recipe-details">
+<div class="details-card">
+<p>Preparation time: <%=@recipe.preparation_time %> </p>
+<p>Cookin time: <%=@recipe.cooking_time %> </p>
+<p>Description: <%=@recipe.description %> </p>
+<%= link_to "Add ingredients",  new_user_recipe_recipe_food_path(recipe_id: @recipe) %>
+<%= link_to "Generate Shopping List", "#"%>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update the styling of the above 2 buttons by putting space between and around. And if possible make them as buttons like from the wireframe to help users know that they are cclickable.